redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -32,22 +31,6 @@ struct Globals {
     backend_ctx: Option<BackendCtx>,
 }
 
-unsafe impl GlobalAlloc for Globals {
```

Comment:
@vityafx I do not think we can drop it without making sure the global allocator is somehow set, otherwise the entire `redisgears_v8_plugin` allocation will not go through Redis allocator and will not be reflected on the stats. On my PR: https://github.com/RedisGears/RedisGears/pull/876 I removed it because I add `redismodule-rs` as a dependency which set the allocator itself.

Reply:
Yes I agree, but the user should be somehow able to provide the allocator for `redisgears_v8_plugin` no?

Reply:
It is today already: https://github.com/RedisGears/RedisGears/blob/34ede8eb1a554fe0cc8efe0c3d052b1ea2789c29/redisgears_plugin_api/src/redisgears_plugin_api/backend_ctx.rs#L32
I guess the conclussion is that we should keep it as I removed it on my PR: https://github.com/RedisGears/RedisGears/pull/876

But maybe we should remove it and re-add it later properly?